### ISSUE TITLE
Missing file 'extras.scss' in v 1.2.64 

### ISSUE THREAD

**Issue by segor** [Templarian/MaterialDesign-Webfont#13]
I have got an error in my project:
```
Error in plugin 'sass'
Message:
    node_modules/mdi/scss/materialdesignicons.scss
  6:9  file to import not found or unreadable: extras
```

**Comment by Templarian** [comment]
For some reason the _extras file didn't get pushed with the changes. Not sure why, looking into it.

**Comment by Templarian** [comment]
Fixed in `v1.2.65` and removed `v1.2.64`.

**Comment by segor** [comment]
Thanks!
